src/comment.js

```diff
@@ -0,0 +1,60 @@
+import API from './api.js';
+
+const api = new API();
+
+export function countComment(res) {
+  if (res.length > 0) {
+    return res.length;
+  }
+  return 0;
+}
```

Comment:
- [ ] Kindly change this and all other regular functions into es6 arrow functions. As required in the [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.) it's important to use es6 syntax.